### PR TITLE
Feature/applet details

### PR DESCRIPTION
**Pull request by vmkhitaryanscn**

### CODE REVIEW

File: src/features/applet-calendar/ui/AppletCalendar.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                  w={45}
+                  h={45}
+                  px={8}
+                  py={8}
+                  br={45}
+                  ai={'center'}
+                  jc={'center'}
+                  bg={bg}>
+                  <Text mb="$1" fontSize={10} color={color}>
+                    {format(date!.timestamp, 'EE').toUpperCase()}
```
Comment (BamMironov) [inline src/features/applet-calendar/ui/AppletCalendar.tsx:44]: Please create a variable for it

File: src/screens/AppletDetailsScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <YStack jc={'flex-start'} flex={1}>
+      <ImageBackground>
+        <AppletCalendar />
+
+        <ScrollView>
+          <XStack mb="$3" py="$2.5" mx="$2" ai="center" bbc="$grey" bbw={1}>
+            <Text color="$darkerGrey">{t('additional:in_progress')}</Text>
+          </XStack>
+
+          <AppletList flex={1} px={14} mb={28} />
```
Comment (BamMironov) [inline src/screens/AppletDetailsScreen.tsx:22]: There shouldn't be AppletList

Comment (BamMironov) [reply src/screens/AppletDetailsScreen.tsx:22]: Please remove ScrollView and everything inside it. Alexey will pass there a widget

**Review by BamMironov — CHANGES_REQUESTED** [review]

File: src/features/applet-calendar/ui/AppletCalendar.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+              <Box h={70}>
+                <YStack
+                  mt={-45}
+                  w={45}
+                  h={45}
+                  px={8}
+                  py={8}
+                  br={45}
+                  ai={'center'}
+                  jc={'center'}
```
Comment (anq83) [inline src/features/applet-calendar/ui/AppletCalendar.tsx:41]: we don't need {} here and above

File: src/features/applet-calendar/ui/AppletCalendar.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+import { CalendarProvider, ExpandableCalendar } from 'react-native-calendars';
+
+import { format } from '@shared/lib/utils';
+import { Text, YStack, Box } from '@shared/ui';
+
+const AppletCalendar: FC = () => {
+  let currentDate = new Date();
+  const weekDay = format(currentDate, 'i');
+  const weekDayNumeric = +weekDay + 1;
```
Comment (anq83) [inline src/features/applet-calendar/ui/AppletCalendar.tsx:11]: +weekDay convert string to number?  just use Number(...)

Comment (anq83) [reply src/features/applet-calendar/ui/AppletCalendar.tsx:11]: Can we avoid data-type in names for the variables that uses in main alghorithm?
Maybe it'd be better to re-name weekDay to weekDayString ?

File: src/features/applet-calendar/ui/AppletCalendar.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+  let firstDay = weekDayNumeric > 7 ? weekDayNumeric - 7 : weekDayNumeric;
```
Comment (anq83) [inline src/features/applet-calendar/ui/AppletCalendar.tsx:13]: how it is possible to get weekDayNumeric > 8?
this code-row looks like workaround

**Review by anq83 — CHANGES_REQUESTED** [review]

**Pull request merged**